src/server/protocol.ts

```diff
@@ -306,6 +306,11 @@ namespace ts.server.protocol {
      */
     export interface DocCommentTemplateRequest extends FileLocationRequest {
         command: CommandTypes.DocCommentTemplate;
+        arguments: DocCommentTemplateRequestArgs;
+    }
+
+    export interface DocCommentTemplateRequestArgs extends FileLocationRequestArgs {
+        generateReturn?: boolean;

```

Comment:
Does this need to be user preference instead ?

Reply:
I thought about that option too, however, the suggested interface https://github.com/microsoft/TypeScript/issues/42637#issuecomment-773092580 relies on arguments

Reply:
I'm fine with either solution. Including the option in the request seemed easier but we could send it in a user preference instead

Reply:
Should the user option be named `generateReturn`?